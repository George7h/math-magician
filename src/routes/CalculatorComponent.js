import React from 'react';
import Calculator from '../components/Calculator';

const CalculatorComponent = () => (
  <div className="calcApp">
    <div>
      <h2>Let&rsquo;s do some math!</h2>
    </div>
    <div>
      <Calculator />
    </div>

  </div>
);

export default CalculatorComponent;
