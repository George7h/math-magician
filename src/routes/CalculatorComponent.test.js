import React from 'react';
import renderer from 'react-test-renderer';
import CalculatorComponent from './CalculatorComponent';

test('Calculator component matches snapshot', () => {
  const tree = renderer.create(<CalculatorComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
