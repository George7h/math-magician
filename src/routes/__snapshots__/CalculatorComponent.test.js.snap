// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator component matches snapshot 1`] = `
<div
  className="calcApp"
>
  <div>
    <h2>
      Let’s do some math!
    </h2>
  </div>
  <div>
    <div
      className="calculator"
    >
      <div
        className="displayScreen"
      >
        <span>
          0
        </span>
      </div>
      <div
        className="btns"
      >
        <button
          className="operator top"
          onClick={[Function]}
          type="button"
        >
          AC
        </button>
        <button
          className="operator top"
          onClick={[Function]}
          type="button"
        >
          +/-
        </button>
        <button
          className="operator top"
          onClick={[Function]}
          type="button"
        >
          %
        </button>
        <button
          className="operator"
          onClick={[Function]}
          type="button"
        >
          ÷
        </button>
        <button
          className="number"
          onClick={[Function]}
          type="button"
        >
          7
        </button>
        <button
          className="number"
          onClick={[Function]}
          type="button"
        >
          8
        </button>
        <button
          className="number"
          onClick={[Function]}
          type="button"
        >
          9
        </button>
        <button
          className="operator"
          onClick={[Function]}
          type="button"
        >
          x
        </button>
        <button
          className="number"
          onClick={[Function]}
          type="button"
        >
          4
        </button>
        <button
          className="number"
          onClick={[Function]}
          type="button"
        >
          5
        </button>
        <button
          className="number"
          onClick={[Function]}
          type="button"
        >
          6
        </button>
        <button
          className="operator"
          onClick={[Function]}
          type="button"
        >
          -
        </button>
        <button
          className="number"
          onClick={[Function]}
          type="button"
        >
          1
        </button>
        <button
          className="number"
          onClick={[Function]}
          type="button"
        >
          2
        </button>
        <button
          className="number"
          onClick={[Function]}
          type="button"
        >
          3
        </button>
        <button
          className="operator"
          onClick={[Function]}
          type="button"
        >
          +
        </button>
        <button
          className="number zero"
          onClick={[Function]}
          type="button"
        >
          0
        </button>
        <button
          className="number"
          onClick={[Function]}
          type="button"
        >
          .
        </button>
        <button
          className="operator"
          onClick={[Function]}
          type="button"
        >
          =
        </button>
      </div>
    </div>
  </div>
</div>
`;
